### PR TITLE
Update after 1370 OS to call xerces and libxml2 init and terminate me…

### PR DESCRIPTION
**Pull request by rosiereflo**
…thods in main

### CODE REVIEW

File: sources/main.cpp [hunk 1]
```diff
@@ -93,6 +95,8 @@ main(int argc, char* argv[]) {
       LOG(error) << MESS(NetworkFileNotFound, runtimeConfig.networkFilePath) << LOG_ENDL;
       return EXIT_FAILURE;
     }
+    xmlInitParser();
+    xercesc::XMLPlatformUtils::Initialize();
```
Comment (lecourtoisflo) [inline sources/main.cpp:99]: Doesn't that depend on a USE_XERCES kind of option on dynawo side ?

File: sources/main.cpp [hunk 2]
```diff
@@ -120,6 +126,8 @@ main(int argc, char* argv[]) {
     LOG(info) << " ============================================================ " << LOG_ENDL;
     LOG(info) << MESS(DFLEnded, context.basename(), timerGlobal.elapsed()) << LOG_ENDL;
 
+    xmlCleanupParser();
```
Comment (lecourtoisflo) [inline sources/main.cpp:129]: Factorize

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

File: sources/main.cpp (hunk 1 above)
Comment (rosiereflo) [reply sources/main.cpp:99]: this is required for all xml parsers beside IIDM

File: sources/main.cpp (hunk 2 above)
Comment (rosiereflo) [reply sources/main.cpp:129]: by that you mean create a method?

Comment (lecourtoisflo) [reply sources/main.cpp:129]: Yes, a method that does both lines, since it is used in multiple location

**Review by gautierbureau — APPROVED** [review]

**Review by lecourtoisflo — APPROVED** [review]

**Pull request merged**